aspnetcore/host-and-deploy/iis/index.md

```diff
@@ -624,6 +575,83 @@ HTTP/2 is enabled by default. Connections fall back to HTTP/1.1 if an HTTP/2 con
 
 For an ASP.NET Core app that targets the .NET Framework, OPTIONS requests aren't passed to the app by default in IIS. To learn how to configure the app's IIS handlers in *web.config* to pass OPTIONS requests, see [Enable cross-origin requests in ASP.NET Web API 2: How CORS Works](/aspnet/web-api/overview/security/enabling-cross-origin-requests-in-web-api#how-cors-works).
 
+::: moniker range=">= aspnetcore-2.2"
+
+## Application Initialization Module and Idle Timeout
+
+When hosted in IIS by the ASP.NET Core Module version 2:
+
+* App's hosted [in-process](xref:fundamentals/servers/index#in-process-hosting-model) or [out-of-process](xref:fundamentals/servers/index#out-of-process-hosting-model), can be configured to start automatically on a worker process restart or server restart.
+* App's hosted [in-process](xref:fundamentals/servers/index#in-process-hosting-model) can be configured not to timeout during periods of inactivity.
```

Comment:
I'm confused about this statement. What specifically are you referring to?

Reply:
Nvrm I see now.

Reply:
I was trying to distinguish from the first point that Idle Timeout only applies to apps hosted in-process. I think these bullet points should be recast with links down to the following two sections to avoid this confusion that you just hit. I'll get it on the next commit.